src/components/Login/Login.css

```diff
@@ -0,0 +1,71 @@
+.overlay {
```

Comment:
If you navigate to a specific page like /obituary before logging in, does that page show up behind the modal still?

Reply:
oh you betcha!

Reply:
so does this mean you can see a user's information before logging in?

Reply:
This is worth double checking. We will need to disable getUser fetch call on Login to be absolutely sure.